app/main.py

```diff
@@ -1,8 +1,46 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int = 7,
```

Comment:
why do you add default value for each argument? delete this here and everywhere. it must be only `comfort_class: int`

Reply:
To maximize profit if we get a broken class instance :)